ntp-proto/src/filter.rs

```diff
@@ -182,16 +185,6 @@ pub struct PeerStatistics {
     pub jitter: f64,
 }
 
-#[allow(dead_code)]
-#[derive(Debug)]
-struct PeerConfiguration {
-    source_address: IpAddr,
-    source_port: u16,
-    destination_address: IpAddr,
-    destination_port: u16,
-    reference_id: ReferenceId,
-}
```

Comment:
Why is this removed?

Reply:
it's unused. I figure we can add it again when we need it (probably when we start working on milestone 2)

Reply:
Ah yes, thought we still used some of it in checkign whether a peer is acceptable, but was mistaken on that.